.github/workflows/pull-requests.yaml

```
@@ -87,14 +87,26 @@ jobs:
         run: echo "SANDBOX_NAME=cozy-e2e-sandbox-$(echo "${GITHUB_REPOSITORY}:${GITHUB_WORKFLOW}:${GITHUB_REF}" | sha256sum | cut -c1-10)" >> $GITHUB_ENV
 
+      - name: Prepare workspace
+        run: |
+          cd ..
+          rm -rf /tmp/$SANDBOX_NAME
+          mv cozystack /tmp/$SANDBOX_NAME
+          sudo systemd-run
+            --on-calendar="$(date -d 'now + 24 hours' '+%Y-%m-%d %H:%M:%S')"
+            --unit=rm-workspace-$SANDBOX_NAME
+            rm -rf /tmp/$SANDBOX_NAME
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)